src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -507,6 +538,10 @@ function getSupportedFeatures() {
         list.push('PlaybackRate');
     }
 
+    if (browser.safari || browser.iOS || browser.iPad) {
```

Comment:
If this is only intended for desktop safari, why the check for iOS and iPad here?